### ISSUE TITLE
window size compatibillity

### ISSUE THREAD

**Issue by sambeard** [raatmarien/playcardswith.me#137]
Not all users will have the same window size, for instance mobile vs. desktop users but also desktop vs. desktop users. Cards can be place at any position and therefore could be out of range for some users. Perhaps the table size should be set beforehand and clients should copy this size, implement it with scrollbars or smth.

**Comment by jchmrt** [comment]
Yeah I think we should define some sort of standard aspect ratio of the table. Then we could simply scale the table on each device to fit the screen.
